docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md

```diff
@@ -670,64 +670,71 @@ paddlepaddle>=2.1
 
 | Operator Name in PaddlePaddle| Limitations|
 | :----------| :----------|
-| adpative_pool2d | The `NHWC` data_layout is not supported. |
 | arg_max | The `int32` output data_type is not supported. |
+| adaptive_pool2d | The `NHWC` data_layout is not supported. |
 | assign |  |
 | assign_value |  |
 | batch_norm |  |
+| bicubic_interp |  |
 | bilinear_interp | `NCW`, `NWC`, `NHWC`, `NCDHW`, `NDHWC` data_layout are not supported. |
-| bilinear_interp_v2 | `NCW`, `NWC`, `NHWC`, `NCDHW`, `NDHWC` data_layout are not supported. |
 | bmm |  |
+| box_coder |  |
 | cast |  |
+| ceil |  |
 | clip |  |
 | concat |  |
+| conditional_block |  |
 | conv2d | `NHWC` data_layout is not supported. |
+| conv2d_transpose |  |
+| cumsum |  |
 | deformable_conv |  |
 | depthwise_conv2d | `NHWC` data_layout is not supported. |
+| depthwise_conv2d_transpose |  |
+| dropout |  |
 | elementwise_add |  |
 | elementwise_div |  |
+| elementwise_floordiv |  |
 | elementwise_max |  |
 | elementwise_min |  |
+| elementwise_mod |  |
 | elementwise_mul |  |
-| elementwise_not_equal |  |
 | elementwise_pow |  |
 | elementwise_sub |  |
 | equal |  |
 | exp |  |
 | expand |  |
-| expand_v2 |  |
 | fill_any_like |  |
 | fill_constant |  |
 | fill_constant_batch_size_like |  |
-| fill_zeros_like |  |
 | flatten_contiguous_range |  |
 | floor |  |
 | gather |  |
-| gather_tree |  |
+| gather_nd |  |
 | gelu |  |
-| generate_proposals_v2 |  |
+| generate_proposals |  |
 | greater_equal |  |
 | greater_than |  |
+| group_norm |  |
 | hard_sigmoid |  |
 | hard_swish |  |
 | layer_norm |  |
 | leaky_relu |  |
 | less_than |  |
+| linear_interp |  |
 | log |  |
 | logical_and |  |
 | logical_not |  |
 | logical_or |  |
 | logical_xor |  |
-| lookup_table_v2 |  |
+| lookup_table |  |
 | matmul |  |
-| matmul_v2 |  |
 | matrix_nms | Only supports IE CPU plugin with *"number of selected boxes"* static shape(e.g.: `min(min(num_boxes, nms_top_k) * num_classes_output, keep_top_k)`). |
 | max_pool2d_with_index |  |
 | meshgrid |  |
-| mul |  |
-| multiclass_nms3 | Only supports IE CPU plugin with *"number of selected boxes"* static shape(e.g.: `min(min(num_boxes, nms_top_k) * num_classes_output, keep_top_k)`). |
+| multiclass_nms | Only supports IE CPU plugin with *"number of selected boxes"* static shape(e.g.: `min(min(num_boxes, nms_top_k) * num_classes_output, keep_top_k)`). |
```

Comment:
```suggestion
| multiclass_nms | Only supports IE CPU plugin with "number of selected boxes" static shape(e.g.: `min(min(num_boxes, nms_top_k) * num_classes_output, keep_top_k)`). |
```
OK thanks for clarifying. I checked in the output, the first two are not working, so I suggest we remove them here.

Reply:
But I can see it works in https://github.com/meiyang-intel/openvino/blob/meiyang/paddle_docs_2022.3/docs/MO_DG/prepare_model/Supported_Frameworks_Layers.md

Reply:
Yeah, some formatting that are displayed correctly on Github might not work with Doxygen. Check it here in the final output: https://docs.openvino.ai/nightly/openvino_docs_MO_DG_prepare_model_Supported_Frameworks_Layers.html#paddlepaddle-supported-operators